src/overview/components/Filters.jsx

```diff
@@ -8,39 +8,82 @@ const Filters = ({
     onShowOnlyBookmarksChange,
     clearAllFilters,
     isClearFilterButtonShown,
+    tagFilterManager,
+    domainFilterManager,
+    onFilterClick,
+    setTagDomainButtonRef,
+    tagFilterPills,
+    domainFilterPills,
 }) => (
```

Comment:
This view is overly complicated.  Could be split up into a number of different shared views and possibly having this view `Filters` just take `props.children` so you can bind in container render methods instead of passing all the individual props.